### DISCUSSION TITLE
Habit tracking

### DISCUSSION THREAD

**Discussion by filcuk** [IgnisDa/ryot#502] [Ideas]
There is currently only one popular self-hosted habit tracker: https://github.com/HabitRPG/habitica.

People often suggest the following Android only app as a great alternative:
https://play.google.com/store/apps/details?id=org.isoron.uhabits

Would there be any interest having this in Ryot?

**Comment by IgnisDa** [comment]
Have you looked into #73? So you think that feature will suffice?
